setup.cfg

```diff
@@ -47,7 +47,7 @@ plugins = pydantic.mypy,sqlmypy
 warn_redundant_casts = True
 warn_unused_ignores = True
 
-[mypy-azafea.tests.*,azafea.event_processors.*.tests.*]
+[mypy-azafea.tests.*,azafea.event_processors.*.tests.*,azafea.event_processors.*.tests_v2.*,azafea.event_processors.*.tests_v3.*]
```

Comment:
That was my commit, and it’s a bit ugly. There must be a better way! (© Raymond Hettinger)

Reply:
We can make it simpler by putting instead ``[mypy-*.tests.*]`` and we should also put the tests_v2 folder in the v2 folder and rename it tests, and same for tests_v3